### ISSUE TITLE
v. 0.9.1.326 - Install-ACMECertificate $certname -Installer iis - conflicting binding

### ISSUE THREAD

**Issue by LubomirH** [ebekker/ACMESharp#314]
Hi, I have spend again some time with your great ACME suite, thanks for it ;)
I however experienced a glitch in the Install-ACMEcertificate command, which prevents me to use it again.
I have used it as it was (being put together 3 months ago), and I am not sure anymore where and how I found out, how it should look like.
There is no parameter -Force for the commandlet self, so I guess by the, in error mentioned, Force parameter is meant the InstallerParameter, which I tried to use.
Can you please confirm that my command syntax is correct?

thanks

L.

```Powershell
PS C:\Windows\system32> Install-ACMECertificate $certname -Installer iis -InstallerParameters @{ WebSiteRef = $iissitename; BindingAddress = '192.168.24.71'; BindingPort = 443
; BindingHost = $DNSname ; CertificateFriendlyName = $DNSname ; force = $true} -Debug
Install-ACMECertificate : found existing conflicting bindings for target site; use Force parameter to overwrite
At line:1 char:1
+ Install-ACMECertificate $certname -Installer iis -InstallerParameters ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-ACMECertificate], InvalidOperationException
    + FullyQualifiedErrorId : System.InvalidOperationException,ACMESharp.POSH.InstallCertificate
```

**Comment by marcel0707** [comment]
I had the same problems. The parameters are case-sensitive. Try -Force instead of -force

**Comment by LubomirH** [comment]
@marcel0707 
you were right ;)

the capital F solved the issue I reported above

'''Powershell
Install-ACMECertificate $certname -Installer iis -InstallerParameters @{ WebSiteRef = $iissitename; BindingAddress = '192.168.24.71'; BindingPort = 443; BindingHost = $DNSname ; CertificateFriendlyName = $DNSname ; sslFlags = 0 ; Force = $true}

It seem that not all the parameters cause such error.
I had to add the sslFlags = 0, because the default seems to be 3, which doesn't work in my case.
And first I entered the parameter with first letter being capital (SslFlags), it didn't work but it didn't crash either.

anyway thanks

L.
